parent/pom.xml

```diff
@@ -866,49 +839,6 @@
         </pluginManagement>
 
         <plugins>
-            <plugin>
-                <!--
-                   Configure Code Coverage measurements...
-                   * Only for Unit tests
-                   * Skips Code Coverage by default
-                   * Generates HTML report
-                   * Output file is target/jacoco.exec
-                   * Output report is target/site/jacoco
-                -->
-                <groupId>org.jacoco</groupId>
-                <artifactId>jacoco-maven-plugin</artifactId>
-                <configuration>
-                    <!--propertyName>jacoco.agent.argLine</propertyName--> <!-- Name it to make the dependecny clearer; default: argLine -->
-                    <skip>${skipCC}</skip>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>pre-unit-test</id>
-                        <goals>
-                            <goal>prepare-agent</goal>
-                        </goals>
-                    </execution>
-                    <execution>
-                        <id>post-unit-test</id>
-                        <phase>test</phase>
-                        <goals>
-                            <goal>report</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <argLine>@{argLine} -Dfile.encoding=UTF-8 -Djava.awt.headless=true</argLine> <!-- activate the JaCoCo javaagent -->
```

Comment:
we might need to leave the plug-in itself, and remove just `@{argLine}`

Reply:
this plugin was restored